README.md

```diff
@@ -1,10 +1,10 @@
 # Data Transfer Project
 
 ## Overview
-The Data Transfer Project makes it easy for users to transfer their data between online services. We provide a common framework and ecosystem to accept contributions from service providers to enable seamless transfer of data into and out of their service.
+The Data Transfer Project makes it easy for people to transfer their data between online services. We provide a common framework and ecosystem to accept contributions from service providers to enable seamless transfer of data into and out of their service.
```

Comment:
This should match the final copy in the upcoming white paper, will update if needed in future PR.

Reply:
thanks, was wondering about that